pkg/services/pluginsintegration/adapters/adapters.go

```diff
@@ -50,9 +50,10 @@ func BackendUserFromSignedInUser(requester identity.Requester) *backend.User {
 		return nil
 	}
 	return &backend.User{
-		Login: requester.GetLogin(),
-		Name:  requester.GetDisplayName(),
-		Email: requester.GetEmail(),
-		Role:  string(requester.GetOrgRole()),
+		Login:           requester.GetLogin(),
+		Name:            requester.GetDisplayName(),
+		Email:           requester.GetEmail(),
+		Role:            string(requester.GetOrgRole()),
+		AuthenticatedBy: requester.GetAuthenticatedBy(),
```

Comment:
nit: I don't agree with giving AuthenticatedBy to every plugin. The backend plugin having an SA capable of querying for the auth_module seems cleaner (as long as it caches so it's not constantly doing it)

It will force us to keep a map of auth modules if we want to make modifications like a general OIDC provider and it has been an internal implementation detail. 

This will probably cause changes to the auth system to break plugins.